tests/unit/models/BucketInfo.js

```diff
@@ -422,6 +435,18 @@ Object.keys(acl).forEach(
                 assert.deepStrictEqual(
                     dummyBucket.getBucketPolicy(), newBucketPolicy);
             });
+            it('setObjectLockConfiguration should set object lock ' +
```

Comment:
Maybe a few extra tests with invalid object lock configurations could make sense too

Reply:
Nevermind, I see the tests in the other PR #996.